### PR TITLE
Add ability to specify the supported NFS protocol versions

### PR DESCRIPTION
**Pull request by votdev**
References:
- https://manpages.debian.org/bullseye/nfs-kernel-server/nfsd.8.en.html
- https://manpages.debian.org/bullseye/nfs-kernel-server/rpc.mountd.8.en.html
- https://wiki.debian.org/NFSServerSetup
- https://www.willhaley.com/blog/ubuntu-nfs-server

- [ ] References issue
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: deb/openmediavault/usr/share/openmediavault/confdb/migrations.d/conf_6.5.0.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# You should have received a copy of the GNU General Public License
+# along with OpenMediaVault. If not, see <http://www.gnu.org/licenses/>.
+
+set -e
+
+. /usr/share/openmediavault/scripts/helper-functions
+
+# Convert the enabled NFS versions into a comma separated list.
+# E.g: +2 -3 +4 -4.1 -4.2 => 2,4
+versions=$(cat /proc/fs/nfsd/versions | sed -E 's/-([[:digit:]](.[[:digit:]])?)//g' | tr -d '+' | sed -e 's/[[:space:]]*$//' | tr -s ' ' ',')
```
Comment (votdev) [inline deb/openmediavault/usr/share/openmediavault/confdb/migrations.d/conf_6.5.0.sh:28]: Any improvement to simplify this conversion is welcome.

Comment (ryecoaaron) [reply deb/openmediavault/usr/share/openmediavault/confdb/migrations.d/conf_6.5.0.sh:28]: sed -e 's/+//g' -e 's/\s/,/g' /proc/fs/nfsd/versions

Comment (votdev) [reply deb/openmediavault/usr/share/openmediavault/confdb/migrations.d/conf_6.5.0.sh:28]: Thx, but the disabled (-X.Y) protocols need to be removed, too.

Comment (ryecoaaron) [reply deb/openmediavault/usr/share/openmediavault/confdb/migrations.d/conf_6.5.0.sh:28]: Ah, I missed that since I don't have any disabled on my system.

Comment (ryecoaaron) [reply deb/openmediavault/usr/share/openmediavault/confdb/migrations.d/conf_6.5.0.sh:28]: This is the simplest I could come up

awk '{ for (i=1; i<=NF; i++) if ($i > 0) printf "%s,", $i }' /proc/fs/nfsd/versions | sed -e 's/+//g' -e 's/,$//'

Comment (votdev) [reply deb/openmediavault/usr/share/openmediavault/confdb/migrations.d/conf_6.5.0.sh:28]: Uff, that also looks not that simple :-)

**Pull request merged**